### PR TITLE
Use Konva groups instead of layers.

### PR DESCRIPTION
**Pull request by badnames**
## Basics

- [x] I added a line to [changelog.md](/doc/changelog.md)
- [x] The PR is rebased with current master.
- [x] Details of what you changed are in commit messages.
- [x] References to issues, e.g. `close #X`, are in the commit messages and changelog.
- [ ] The buildserver is happy.

## Checklist

- [ ] I have installed and I am using [pre-commit hooks](../doc/contrib/README.md#Hooks)
- [ ] I fully described what my PR does in the documentation
- [ ] I fixed all affected documentation
- [ ] I fixed the introduction tour
- [ ] I wrote migrations in a way that they are compatible with already present data
- [ ] I fixed all affected decisions
- [ ] I added automated tests or a [manual test protocol](../doc/tests/manual/protocol.md)
- [ ] I added code comments, logging, and assertions as appropriate
- [ ] I translated all strings visible to the user
- [ ] I mentioned [every code or binary](https://github.com/ElektraInitiative/PermaplanT/blob/master/.reuse/dep5) not directly written or done by me in [reuse syntax](https://reuse.software/)
- [ ] I created left-over issues for things that are still to be done
- [ ] Code is conforming to [our Architecture](/doc/architecture)
- [ ] Code is conforming to [our Guidelines](/doc/guidelines)
- [ ] Code is consistent to [our Design Decisions](/doc/decisions)
- [ ] Exceptions to any guidelines are documented

## Review

- [ ] I've tested the code
- [ ] I've read through the whole code
- [ ] I've read through the whole documentation
- [ ] I've checked conformity to guidelines

### CODE REVIEW

**Comment by badnames** [comment]
jenkins build please

File: frontend/src/features/map_planning/utils/ShapesSelection.ts [hunk 1]
```diff
@@ -23,10 +26,12 @@ export const selectIntersectingShapes = (
   if (stageRef.current.children === null) return;
 
   // we don't always have to look for them, we can store them
-  const allShapes = stageRef.current.children
+  const allShapes = stageRef.current.children?.[0].children
     //filter out layers which are not selected
     ?.filter((layer) => layer.attrs.listening)
-    .flatMap((layer) => layer.children)
+    // All PermaplanT layers are displayed as a single Konva layer.
+    // It might be possible to
```
Comment (markus2330) [inline frontend/src/features/map_planning/utils/ShapesSelection.ts:33]: Sentence broken?

**Review by markus2330 — APPROVED** [review]
CI fails. Probably better to merge this after release, as it received no testing yet?

**Comment by badnames** [comment]
jenkins build please

**Comment by markus2330** [comment]
@4ydan do you know why this e2e test fails?

**Comment by 4ydan** [comment]
It looks like the whole page crashed on the test.

**Comment by markus2330** [comment]
It is not release critical, so let us debug this later.

**Comment by markus2330** [comment]
@badnames pls rebase and set as ready to merge

**Comment by markus2330** [comment]
The cursor layer needs to be commented out, otherwise e2e will fail.

**Comment by markus2330** [comment]
Unfortunately still fails.

**Comment by badnames** [comment]
#1224 is the updated branch

**Pull request closed without merging**